### ISSUE TITLE
Agricraft Support

### ISSUE THREAD

**Issue by homemmakako** [BlakeBr0/MysticalAgriculture#12]
Since agricraft has been updated for 1.10.2, can you make so you can put MA seeds in crop sticks (from agricraft)?

**Comment by BlakeBr0** [comment]
Someone would have to go through and manually create jsons for all the crops. 
I'm not going to simply because I don't really want to, but if someone else does then there you go.

**Comment by homemmakako** [comment]
I actually tried to do that. I had, however, a bunch of problems. For instante, I did manage to configure inferium seeds tier 1, but couldn't do the same with aluminum seeds using the same code (with the apropriate ajustments, of course). Maybe I Just made one (or many) silly mistakes, but the thing is: I'm not a professional, all I do is follow the default config and change the keywords as any other noob, so I am never able to easily identify bugs. If someone was actually able configure those dam seeds, I would very much apreciate a copy of your configs ;P
